lib/internal/Magento/Framework/Stdlib/DateTime/Timezone.php

```diff
@@ -246,13 +247,13 @@ public function isScopeDateInInterval($scope, $dateFrom = null, $dateTo = null)
             // fix date YYYY-MM-DD 00:00:00 to YYYY-MM-DD 23:59:59
             $toTimeStamp += 86400;
         }
+        $result = true;
 
-        $result = false;
-        if (!$this->_dateTime->isEmptyDate($dateFrom) && $scopeTimeStamp < $fromTimeStamp) {
-        } elseif (!$this->_dateTime->isEmptyDate($dateTo) && $scopeTimeStamp > $toTimeStamp) {
-        } else {
-            $result = true;
+        if (!$this->_dateTime->isEmptyDate($dateFrom) && $scopeTimeStamp < $fromTimeStamp ||
+            !$this->_dateTime->isEmptyDate($dateTo) && $scopeTimeStamp > $toTimeStamp) {
+            $result = false;
         }
+
         return $result;
```

Comment:
Please refactor it into a single predicate.

Reply:
@orlangur remove empty line ?

Reply:
Or did you mean set` return true; or false; `on if statements ?

Reply:
Do it as `return <identical condition>;`

Reply:
@orlangur  done :heavy_check_mark: